sse.go

```diff
@@ -0,0 +1,144 @@
+// Copyright 2022 Flamego. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package sse
+
+import (
+	"encoding/json"
+	"net/http"
+	"reflect"
+	"time"
+
+	"github.com/flamego/flamego"
+)
+
+const (
+	defaultPingInterval = 10 * time.Second
+)
```

Comment:
If this is private _and_ only used once, I don't see a reason to make it a package-level constant.